src/test/java/org/cactoos/scalar/AndInThreadsTest.java

```diff
@@ -83,22 +82,24 @@ public void allFalse() throws Exception {
                     new False(),
                     new False()
                 )
-            ).value(),
-            Matchers.equalTo(false)
+            ),
+            new ScalarHasValue<>(false)
         );
     }
 
     @Test
     public void emptyIterator() throws Exception {
         MatcherAssert.assertThat(
-            new AndInThreads(Collections.emptyList()).value(),
-            Matchers.equalTo(true)
+            new AndInThreads(new IterableOf<Scalar<Boolean>>()),
+            new ScalarHasValue<>(true)
         );
     }
 
     @Test
     public void iteratesList() {
-        final List<String> list = new LinkedList<>();
+        final List<String> list = Collections.synchronizedList(
+            new ArrayList<String>(2)
+        );
```

Comment:
@paulodamaso I thought about it but `SyncCollection` does not implement `add` method.

Reply:
@pbenety Right, it needs to be mutable for the test, so that's ok for now. Just open a `@todo` about removing these static methods access from here so we can think about it in the future.

Reply:
@paulodamaso @todo added.